testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/realm/LoginSettingsTest.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.keycloak.testsuite.console.realm;
 
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
```

Comment:
Please remove the imports. You've already imported it by the static imports below.

Reply:
yes sorry !